bliss/train.py

```diff
@@ -32,7 +32,7 @@ def train(cfg: DictConfig):
     dataset = instantiate(cfg.training.dataset)
 
     # setup model
-    model = instantiate(cfg.training.model, optimizer_params=cfg.training.optimizer_params)
+    model = instantiate(cfg.training.model)
```

Comment:
It seems simpler to include the optimizer parameter in `cfg.training.model`, particularly now that I've introduced parameters for the scheduler too. But please lmk if I'm missing something.